src/include/checkpoint.h

```diff
@@ -10,26 +10,29 @@
 
 #define WT_CHECKPOINT_SESSION_FLAGS (WT_SESSION_CAN_WAIT | WT_SESSION_IGNORE_CACHE_SIZE)
 
-#define WT_CHECKPOINT_STATE_INACTIVE 0
-#define WT_CHECKPOINT_STATE_APPLY_META 1
-#define WT_CHECKPOINT_STATE_APPLY_BTREE 2
-#define WT_CHECKPOINT_STATE_UPDATE_OLDEST 3
-#define WT_CHECKPOINT_STATE_SYNC_FILE 4
-#define WT_CHECKPOINT_STATE_EVICT_FILE 5
-#define WT_CHECKPOINT_STATE_BM_SYNC 6
-#define WT_CHECKPOINT_STATE_RESOLVE 7
-#define WT_CHECKPOINT_STATE_POSTPROCESS 8
-#define WT_CHECKPOINT_STATE_HS 9
-#define WT_CHECKPOINT_STATE_HS_SYNC 10
-#define WT_CHECKPOINT_STATE_COMMIT 11
-#define WT_CHECKPOINT_STATE_META_CKPT 12
-#define WT_CHECKPOINT_STATE_META_SYNC 13
-#define WT_CHECKPOINT_STATE_ROLLBACK 14
-#define WT_CHECKPOINT_STATE_LOG 15
-#define WT_CHECKPOINT_STATE_RUNNING 16
-#define WT_CHECKPOINT_STATE_ESTABLISH 17
-#define WT_CHECKPOINT_STATE_START_TXN 18
-#define WT_CHECKPOINT_STATE_CKPT_TREE 19
+/* Checkpoint state ordered by appearance in txn_ckpt.c. Inactive should always be 0. */
```

Comment:
The first sentence makes it sound like the order matters - how about something more like `Inactive should always be 0. Other states happen to be roughly ordered by appearance in txn_ckpt.c.`

Reply:
I don't think we should mention any file. Show we more say something about the checkpoint lifecyle?

Reply:
I like it. @korteland?

Reply:
That's even better than what I suggested. Thanks!

Reply:
comment updated here :[e5e713e](https://github.com/wiredtiger/wiredtiger/pull/10733/commits/e5e713e5402219959644f91e669bc0e2c9276cd8)